src/handlers/signup.js

```diff
@@ -13,7 +13,10 @@ const GITHUB_API_URL = 'https://api.github.com'
 
 module.exports = function signup(message) {
   try {
-    const [username, platforms] = parseSignup(message.content)
+    const [username, platforms] = parseSignup(
+      message.content,
+      message.member.id,
```

Comment:
Shouldn't this be `.author.id`? Or can we use member as well?

Reply:
We can use 'member' as well, given that it will return the same _as long as the user is also a member_ of the guild where the bot is listening. `.member.id` is read-only while `.author.id` is not.

Reference: https://discord.js.org/#/docs/main/stable/class/Message?scrollTo=member